assets/js/pages/Login/LoginSSO.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import Button from '@common/Button';
+
+export default function LoginSSO({ singleSignOnUrl }) {
+  return (
+    <Button
+      onClick={() => {
+        window.location.href = singleSignOnUrl;
```

Comment:
We are not using a child href for styling purposes?

Reply:
You mean to have `<a href={ singleSignOnUrl }>Login with Single Sign-on</a>` as child, right?
This has the difference that the url is visible in the browser, when you put the mouse on the button.
I don't have any preference. I have seen the `window.location.href` in other login/session react component in our code. That's why I used it

Reply:
And yes, the button is what we need related to styling, so it was an "obvious" option